### PR TITLE
Save labels and other related documents on tracking number

### PR DESCRIPTION
**Pull request by bhavana94**

### CODE REVIEW

File: stock.py [hunk 1]
```diff
@@ -447,7 +447,8 @@ def generate_shipping_labels(self, **kwargs):
                     stock_package.code,
                 ),
                 'data': buffer(base64.decodestring(data)),
-                'resource': '%s,%s' % (self.__name__, self.id)
+                'resource':
+                '%s,%s' % (self.tracking_number.__name__, self.tracking.id)
```
Comment (uzayr) [inline stock.py]: change it to 'resource': '%s,%d' % (self.tracking_number.__name__, self.tracking.id)

Comment (bhavana94) [inline stock.py]: that leads to flake8 issue

**Comment by riteshshrv** [comment]
LGTM

File: stock.py [hunk 2]
```diff
@@ -447,7 +447,10 @@ def generate_shipping_labels(self, **kwargs):
                     stock_package.code,
                 ),
                 'data': buffer(base64.decodestring(data)),
-                'resource': '%s,%s' % (self.__name__, self.id)
+                'resource':
+                '%s,%d' % (
+                    self.tracking_number.__name__, self.tracking_number.id
```
Comment (tarunbhardwaj) [inline stock.py]: Tracking number is added to self in line 461.

File: stock.py [hunk 3]
```diff
@@ -447,7 +447,10 @@ def generate_shipping_labels(self, **kwargs):
                     stock_package.code,
                 ),
                 'data': buffer(base64.decodestring(data)),
-                'resource': '%s,%s' % (self.__name__, self.id)
+                'resource':
+                '%s,%d' % (
+                    self.tracking_number.__name__, self.id
```
Comment (riteshshrv) [inline stock.py]: `self.tracking_number` doesn't exist here; its created on L456. So, please move this block below

**Pull request closed without merging**